index.bs

```diff
@@ -5929,10 +5891,10 @@ partial interface MLGraphBuilder {
 <dl dfn-type=dict-member dfn-for=MLTriangularOptions>
     : <dfn>upper</dfn>
     ::
-        A {{boolean}} value. Indicate whether the output the upper or the lower part of the input matrix is retained. If not set, it is assumed to be true, indicating that the upper part is retained.
+        Indicates whether the output the upper or the lower part of the input matrix is retained. True indicates that the upper part is retained.
     : <dfn>diagonal</dfn>
     ::
-        A {{long}} value. Specify how many diagonals above or below the main diagonals of the input matrix are retained or excluded. If not set, this value is assumed to be 0, which means no diagonals other than the main diagonals are affected. 
+        How many diagonals above or below the main diagonals of the input matrix are retained or excluded. A value of 0 means no diagonals other than the main diagonals are affected.
```

Comment:
Should we keep "Specify" here? Maybe changes it "Specifies". Otherwise, "How many" sounds like a question sentence, at leas to me?

Reply:
updated!